benchmarks/counter.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
introduced benchmarks for counter (same code as `gauge` benchmarks)

Reply:
would you mind sending a separate PR with this while we figure out whether or not to land this? Very useful on its own 🙂

Reply:
created pr - https://github.com/siimon/prom-client/pull/596 🙏

Reply:
thanks!